packages/frontend-2/middleware/headers.global.ts

```diff
@@ -0,0 +1,17 @@
+export default defineNuxtRouteMiddleware((to) => {
+  const { ssrContext } = useNuxtApp()
+
+  if (ssrContext) {
+    // Add response header that shows this is a FE2 request
+    ssrContext.event.node.res.setHeader('x-speckle-frontend-2', 'true')
+
+    // Check if the route is not an possible embedded route
+    // If not add the header to prevent click highjacking
+    if (to.name !== 'projects-id-models-modelId') {
```

Comment:
@Mikehrn minor issue, but it would be better to set a static name for the route in the page's `definePageMeta` instead of relying on this autogenerated one, which could theoretically change.